prov/util/src/util_ep.c

```diff
@@ -58,3 +58,13 @@ int ofi_endpoint_init(struct fid_domain *domain, const struct util_prov *util_pr
 	atomic_inc(&util_domain->ref);
 	return 0;
 }
+
+int ofi_endpoint_close(struct util_ep *util_ep)
+{
+	struct util_domain *util_domain;
+
+	util_domain = util_ep->domain;
+	atomic_dec(&util_domain->ref);
```

Comment:
Thanks. There is no need to declare `util_domain` variable. We can decrement the ref directly:
`atomic_dec(&util_ep->domain->ref);`